### ISSUE TITLE
Getting inner SVG content

### ISSUE THREAD

**Issue by wiiiteek** [svgdotjs/svg.js#700]
Is there any way to get inner content of SVG draw?

Method `draw.svg();` returns whole svg including `<svg></svg>` tags but is there any way to get content between those tags?

For now I am using workaround `draw.node.innerHTML` to fetch inner content, but I am not sure if it's right way.

**Comment by Fuzzyma** [comment]
No there is no Way. Only thing I can think of is looping through the children and call svg on each one.

**Comment by wiiiteek** [comment]
OK, I see.

I am afraid that looping through children can take too much CPU. Especially that in my program I am checking SVG content every 10sec to check if content has changed.

I think I am just gonna parse it through this regexp `/<svg[^>]*>(.*)<\/svg>/s`

Maybe this is some idea for a new feature? :)

**Comment by Fuzzyma** [comment]
Already on the list ;)

**Comment by saivan** [comment]
What was the method we were going to use to achieve this? Is the loop still the way to go?

**Comment by Fuzzyma** [comment]
Didndt think it trough yet. Maybe an additional parameter for `svg()` or a new function `innerSVG()` or so?

**Comment by saivan** [comment]
Well maybe yep `svg(wrapped=false)` is probably the best syntax wise. But is this easy to implement?

**Comment by Fuzzyma** [comment]
Problem is that svg is a setter and a getter. But yes when you check the type of the argument it would work. I also thought about making passing a function possible which is called for every element which is exported. You could even do the rounding for numbers there to shrink the svg size

**Comment by Fuzzyma** [comment]
This was implemented with 38ce7a752078588aca394ef91f8c80502e68731a.
The following is now possible:

```js
// signatures
element.svg(modifier, outerHtml) // or
element.svg(outerHtml)

// outerHtml defaults to false when importing and to true when importing

element.svg('<someElement>') // imports content into element
element.svg('<someElement>', true) // replaces the node with the content

element.svg() // returns the element with content
element.svg(false) // returns only the content without the element wrapped around it

// with export modifier
// the modifier is called for every element which is exported
element.svg(function(element) {
  return element// replaces the current instance with anotherone
  return false // deletes the element so its not exported

  element.attr('x', 500) // just change the element before export
})

// this ofc can also be called with outerHtml=false
```

**Comment by Fuzzyma** [comment]
One note: When importing elements and outerHTML beeing true, the current elemtent is replaced with the imported content. That means, that we dont have a `this` to return. Thats why, whenever importing with outerHTML beeing true, the parentNode is returned instead.
